src/ResourceManagement/Media/Media.Tests/Helpers/RecordedDelegatingHandler.cs

```diff
@@ -0,0 +1,102 @@
+﻿//
```

Comment:
This class is unnecessary and is adding to the code for your tests unnecessarily.  Since you are always usinbg PassThrough = true, there is no need for this at all.  The expected status code is checked in the generated code, or an exception should be returned.  This class is only really useful if you want to test scenarios that are difficult to capture life and have to synthesize service responses.  Please remove this class and its use for now.